identity-access-manager-keycloak/docker-compose.dev.yml

```diff
@@ -0,0 +1,8 @@
+version: '3.9'
+
+services:
+  identity-access-manager-keycloak:
+    ports:
+      - target: 8080
+        published: 9080
```

Comment:
Can you add this port to the exposed port env var in nginx please in package-metdata.json

Reply:
It might work in insecure mode (with a different config), but I thought it didn't make sense to add it that way, we are adding keycloak as a security enhancement and I don't think we should support insecure deployments going forward. I will discuss this with Ryan, but for now, we don't need it, so no point in spending time on it.

Reply:
okay cool :)